job/runner.go

```diff
@@ -139,7 +140,15 @@ func (j *JobRunner) RemoteRun() (string, error) {
 
 	// Normalize the method passed by the user
 	method := strings.ToUpper(j.job.RemoteProperties.Method)
-	bodyBuffer := bytes.NewBufferString(body)
+	bodyBuffer := &bytes.Buffer{}
+	mime := j.job.RemoteProperties.Headers.Get("Content-Type")
+	if mimeFormData == mime || mimeFormURLEncoded == mime {
```

Comment:
This seems like the key part of the PR, here.

My question is: since you can pass whatever value you want as the body of the POST request, why not just encode it properly to begin with?  Why is there specific magic needed in Kala for this?

I understand that the body of the request may be larger since, I gather, it's binary nestled in JSON.  However:
1. How large are we talking?  (As just one data point, for *your* specific use case what's the filesize?)
2. Should Kala be used to send large files?  Is this a correct use of it?

Reply:
We're are building something similar to Segment.com, that leads us to so many heavy job which transfer a lot of Data based on Send S3 file to some destination.
I think sending file is really important.
but right now! those implementation are not ready yet!